schema/bom-1.6.proto

```diff
@@ -726,6 +726,14 @@ message EvidenceOccurrences {
   optional string bom_ref = 1;
   // The location or path to where the component was found.
   string location = 2;
+  // The line number where the component was found.
+  optional int32 lineNumber = 3;
```

Comment:
We use line and column in frames. Can we use the same keys?

Reply:
Good point @prabhu. I'll make the changes to rename `lineNumber` to `line`.

@bhess Any objection to changing `offset` to `column`?

Reply:
The main use case we had were binaries where crypto assets occur at a given offset of the full binary. Column seems to be relative to a line, right? Not sure if this is the same then. Is an option to include both `offset` and `column`?

Reply:
Ah ok. That's what I thought, but wanted to better understand. Thanks for clarifying. We'll leave offset in place then.